test/tap/search.js

```diff
@@ -1,84 +1,185 @@
-var fs = require('graceful-fs')
 var path = require('path')
```

Comment:
Do any of the tests in here match against the description field? Or username fields? These are both things that I've found surprisingly useful in the past, when search worked. I'm pretty sure they work in practice (testing right now), but it would be good to have them included in the tests.